JapaneseCalendar/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace JapaneseCalendar
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                string str;
+                Console.WriteLine("Input the number of the year (Press Enter to stop the program)");
+                str = Console.ReadLine();
+                if (str == string.Empty)
+                    break;
+                var year = int.Parse(str);
+                if (!int.TryParse(str, out year))
+                {
+                    Console.WriteLine("Input error. ");
+                    continue;
+                }
+
+                int numberOfYear;
+                if (year >= 1984)
+                    numberOfYear = (year - 1984) % 60;
+                else
+                    numberOfYear = (year - 1984) % 60 + 60;
+                Console.WriteLine("{0} is the year of {1} {2}.", year, getColour(numberOfYear / 12), getAnimal(numberOfYear % 12));
```

Comment:
Хорошо бы поправить (мою) ошибку
```suggestion
                Console.WriteLine("{0} is the year of a {1} {2}.", year, getColour(numberOfYear / 12), getAnimal(numberOfYear % 12));
```